packages/calling-stateful-client/src/StatefulCallClient.ts

```diff
@@ -139,7 +150,11 @@ export interface StatefulCallClient extends CallClient {
    * @param options - The CallAgentOptions for additional options like display name.
    * @public
    */
-  createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
+  createCallAgent(
```

Comment:
Why are we completely changing the function signature here?
Would `createCallAgent(...args: Parameters<CallClient['createCallAgent']>, alternativeCallerId?: string)` not work?

Reply:
this unfortunately didn't work tsc hated the idea of a optional prop before a required one, and as well it hated the spread of `...args` happening before the `alternativeCallerId`